internal/config/yaml_parser.go

```diff
@@ -49,6 +51,16 @@ func IsValidLogSeverity(severity LogSeverity) bool {
 	return false
 }
 
+func IsValidLogRotateConfig(config LogRotateConfig) error {
+	if config.MaxFileSizeMB <= 0 {
+		return fmt.Errorf("max-file-size-mb cannot be less than 0")
```

Comment:
nit: less than or equal to 0. At both places.

Reply:
Changed to more than 1 at both places to also take care of floating point values as mentioned in the other comment.